examples/petsc/multigrid.c

```diff
@@ -318,6 +318,7 @@ int main(int argc, char **argv) {
 
     // Multiplicity
     CeedElemRestrictionGetMultiplicity(ceeddata[i]->Erestrictu,
+                                       CEED_NOTRANSPOSE,
```

Comment:
`CEED_NOTRANSPOSE` argument is no longer part of this function

Reply:
Ah, the proposed rebase got this

Reply:
Thanks. I'd already fixed this in my merge branch (since it's a semantic conflict). Should we be reviewing there?

Reply:
Ok. You can cherry-pick the small commits I've done here then

Reply:
Nah, your changes are fine. I'd planned to repeat the merge. (Git has a clever feature called [rerere](https://git-scm.com/docs/git-rerere) that makes such things manageable.)